core/src/scene/sceneLoader.cpp

```diff
@@ -37,10 +40,15 @@ void SceneLoader::loadScene(const std::string& _file, Scene& _scene, TileManager
     loadCameras(config["cameras"], _view);
     loadLights(config["lights"], _scene);
 
-    for (auto& style : _scene.getStyles()) {
-        style->build(_scene.getLights());
+    for (auto& style : _scene.styles()) {
+        style->build(_scene.lights());
     }
 
+    // Styles that are opaque must be ordered first in the scene so that they are rendered 'under' styles that require blending
+    std::sort(_scene.styles().begin(), _scene.styles().end(), [](std::unique_ptr<Style>& a, std::unique_ptr<Style>& b) {
```

Comment:
There is also [this](http://en.cppreference.com/w/cpp/experimental/parallelism/existing#sort).. But "experimental" :|